src/visitor.js

```diff
@@ -134,8 +134,7 @@ export const visitElement = (
     case REACT_MEMO_TYPE: {
       const memoElement = ((element: any): MemoElement)
       const type = memoElement.type.type
-      const fauxElement = (createElement((type: any), memoElement.props): any)
-      const child = render(type, memoElement.props, queue, visitor, fauxElement)
```

Comment:
This is obviously a very small change, but to leave a comprehensive explanation in this PR:
The `fauxElement` was creating a the `React.Element` correctly but was assuming it can only be an element of a function or class component, hence it immediately jumps into `render`. The alternative is to just return the `React.Element`, not assume anything about it, and let the `visitElement` visitor run over it again to check what kind of element it really is